lib/bloom.js

```diff
@@ -32,7 +33,7 @@ var Bloom = module.exports = function (bitvector) {
 /**
  * adds an element to a bit vector of a 64 byte bloom filter
  * @method add
- * @param {Buffer} element
+ * @param e - the element
```

Comment:
I am not really getting the change here. This calls ``dropLeadingZeroes()`` which requires a Buffer.

Reply:
Yup, should've kept the type, I'll add that back. The main change is the param name which was incorrect (i.e. `s/element/e/`).